ts/features/itwallet/machine/credential/selectors.ts

```diff
@@ -25,3 +26,8 @@ export const selectCredentialOption = (snapshot: MachineSnapshot) =>
 
 export const selectFailureOption = (snapshot: MachineSnapshot) =>
   O.fromNullable(snapshot.context.failure);
+
+export const selectCredential = (snapshot: MachineSnapshot) =>
+  pipe(O.fromNullable(snapshot.context.credential), credential =>
+    O.isSome(credential) ? credential.value : undefined
+  );
```

Comment:
I don't understant: this functions takes a nullable, transforms it to an `Option` and then returns a nullable. It is the same as:

```suggestion
export const selectCredential = (snapshot: MachineSnapshot) =>
  snapshot.context.credential;
```
Am I missing something? Couldn't we just use the `selectCredentialOption` selector?